core/config/project_settings.h

```diff
@@ -104,6 +104,8 @@ class ProjectSettings : public Object {
 
 	HashMap<StringName, AutoloadInfo> autoloads;
 
+	HashMap<StringName, String> global_groups;
```

Comment:
```suggestion
	HashMap<StringName, String> global_groups;
```
I think the separation is unnecessary here.